book/src/csi-proxy.md

```diff
@@ -0,0 +1,49 @@
+# CSI Proxy
+
+## Status and Releases
+
+**Git Repository:** [https://github.com/kubernetes-csi/csi-proxy](https://github.com/kubernetes-csi/csi-proxy)
+
+**Status:** V1 starting with v1.0.0
+
+Status | Min K8s Version | Max K8s Version
+--|--|--
+v0.1.0 | 1.18 | - 
+v0.2.0+ | 1.18  | -
+v1.0.0+ | 1.18  | -
+
+## Description
+
+CSI Proxy is a binary that exposes a set of gRPC APIs around storage operations over named pipes in Windows. A container, such as CSI node plugins, can mount the named pipes depending on operations it wants to exercise on the host and invoke the APIs.
+
+Each named pipe will support a specific version of an API (e.g. v1alpha1, v2beta1) that targets a specific area of storage (e.g. disk, volume, file, SMB, iSCSI). For example, `\\.\pipe\csi-proxy-filesystem-v1alpha1`, `\\.\pipe\csi-proxy-disk-v1beta1`. Any release of csi-proxy.exe binary will strive to maintain backward compatibility across as many prior stable versions of an API group as possible. Please see details in this [CSI Windows support KEP](https://github.com/kubernetes/enhancements/tree/master/keps/sig-windows/1122-windows-csi-support)
+
+## Usage
+
+Run csi-proxy.exe binary directly on a Window node. Starting from v1.0.0, the command line options are
+
+* -kubelet-path: This is the prefix path of the kubelet directory in the host file system
+
+Prior v1.0.0, two command line options were used:
+
+* `-kubelet-csi-plugins-path`: This is the prefix path of the Kubelet plugin directory in the host file system (`C:\var\lib\kubelet` is used by default).
+
+* `-kubelet-pod-path`: This is the prefix path of the kubelet pod directory in the host file system (`C:\var\lib\kubelet` is used by default).
+
+* `-windows-service`: Configure as a Windows Service
```

Comment:
Are these options also available in 1.0? The indenting makes it seem like these options are not available in 1.0 anymore. Maybe for simplicilty, let's remove the detailed entries for the old kubelet path arguments, and just mention in the description for `kubelet-path` that it replaces `kubelet-pod-path` and `kubelet-csi-plugins-path` in previous versions.